docs/elements.md

```diff
@@ -1577,6 +1578,36 @@ Attribute | Type | Default | Description
 - [demo/autograder/python/plots]
 - [demo/autograder/python/random]
 
+### `pl-xss-safe` element
+
+Removes potentially dangerous scripts from an HTML code. This is recommended when parsing and displaying student-provided code. Note that any code parsed by this element must be supported directly by the browser, i.e., it cannot include PrairieLearn elements or special tags.
+
+#### Sample element
+
+```html
+<pl-xss-safe>
+{{{submitted_answers.answer}}}
```

Comment:
What will happen if `submitted_answers.answer` contains an unmatched `</pl-xss-safe>` followed by some dangerous code?

Reply:
That's a great point. Probably good to only get input from files or feedback fields directly. I'll change it to account for that.

Reply:
Maybe allow for something like `<pl-xss-safe content="{{submitted_answers.answer}}"></pl-xss-safe>`, since that would ensure the case above never happens?

Reply:
Is mustache quoting guaranteed to be safe for use in an attribute value? I thought it's quoting rules were only safe for HTML? I'm thinking about someone putting `"` characters in `submitted_answers.answer`, for example.

Reply:
Yes, it replaced that with &quot;

Reply:
Ok, sounds good to me then.